versioned/dynamodb/src/main/java/com/dremio/nessie/versioned/store/dynamo/DynamoGeneralReadFailure.java

```diff
@@ -15,7 +15,7 @@
  */
 package com.dremio.nessie.versioned.store.dynamo;
 
-class DynamoGeneralReadFailure extends RuntimeException {
+public class DynamoGeneralReadFailure extends RuntimeException {
```

Comment:
Why this change?

Reply:
Not sure. I think my IntelliJ may have complained. I'll revert and check it still builds.